var/spack/repos/builtin/packages/cdo/package.py

```diff
@@ -66,7 +70,12 @@ class Cdo(AutotoolsPackage):
 
     depends_on('udunits', when='+udunits2')
     depends_on('libxml2', when='+libxml2')
-    depends_on('proj@:5', when='+proj')
+
+    # PROJ.6 supported starting 1.9.8.1
+    # https://launchpad.net/ubuntu/+source/cdo/+changelog
```

Comment:
Officially, there is no version `1.9.8.1`: https://code.mpimet.mpg.de/projects/cdo/files
Could you, please, replace the reference link with https://code.mpimet.mpg.de/projects/cdo/news?